src/Parameters.jl

```diff
@@ -0,0 +1,23 @@
+module Parameters
+
+abstract type AbstractInsolationParams end
+const APS = AbstractInsolationParams
+
+Base.@kwdef struct InsolationParameters{FT} <: AbstractInsolationParams
+    obliq_epoch::FT
+    lon_perihelion_epoch::FT
+    day::FT
+    orbit_semimaj::FT
+    eccentricity_epoch::FT
+    tot_solar_irrad::FT
+    year_anom::FT
+    mean_anom_epoch::FT
+    epoch::FT
```

Comment:
What I would like to do is have this be a String. I’m not currently using the `epoch` value from CLIMAParameters because Ollie said it was too difficult to change the default to be a String. (See ZenithAngleCalc.jl where I have `epoch_string`.) I’m not sure how to do it exactly, but you’ll have to update the `create_parameters.jl` file to not just take the defaults from CLIMAParameters. Honestly for the other orbital parameters it might just be worth having those local to this package as well since nothing else in the code should need to know the eccentricity of the orbit (for example). But if this is tricky for some reason, we can change it in a future PR when I’m back!

Reply:
Right, the string bit is a bit difficult at the moment. Let's tackle this in a later PR, since this one is already pretty busy.